### DISCUSSION TITLE
Wait for automation to complete

### DISCUSSION THREAD

**Discussion by tedr56** [Budibase/budibase#12331] [Help] [answered]
Hi all.
Is there any to wait for automation to complete in actions?
I tried [this](https://github.com/Budibase/budibase/discussions/11277#discussioncomment-6495179) but doesn't seem to work.

I have an automation that query some api, and I need to wait the answers to be recorded in DB to show the result.
Is it possible?

Thanks

**Comment by shogunpurple** [comment, marked as answer]
https://docs.budibase.com/docs/synchronous-automations
